toolbox/util/githubClient.go

```diff
@@ -105,13 +110,31 @@ func (g GithubClient) CreatePullRequest(
 // AddAutoMergeLabelsToPR adds /lgtm and /approve labels to a PR,
 // essentially automatically merges the PR without review, if PR passes presubmit
 func (g GithubClient) AddAutoMergeLabelsToPR(repo string, pr *github.PullRequest) error {
-	labels := []string{
-		"lgtm",
-		"approved",
-		"release-note-none",
-	}
 	_, _, err := g.client.Issues.AddLabelsToIssue(
-		context.Background(), g.owner, repo, pr.GetNumber(), labels)
+		context.Background(), g.owner, repo, pr.GetNumber(), autoMergeLabels)
+	return err
+}
+
+func (g GithubClient) ApproveAutoPR(repo string, pr *github.PullRequest) error {
```

Comment:
This is cool, but you cannot approve your own PR. You would need another bot with admin priviledge to approve it. I would advise to create a new label like review-not-required, and auto approve all PRs with that label.

Reply:
@sebastienvas @yutongz How about we initialize another githubClient right here with the token of another bot/identity so the pr we just created gets approved right away? I am worried about timing issues if we were to have another bot to approve auto PRs asynchronously, since we assume any PR that remains pending/open for an extended period has failed presubmit and we close them after that.